BoundingBox bbox
string class_id